### ISSUE TITLE
SFTP failure to connect

### ISSUE THREAD

**Issue by wnjohnson3** [iterate-ch/cyberduck#16368]
I’ve been using cyberduck for years but recently (new mac m1 sonoma) I can not use cyberduck to sftp connect to anything.  So I purchased 9.0.2 from the Apple Store and still having the same problem.

Most recent testing is connection to 2nd account on my mac.

command line sftp test@192.168.146.200 works just fine,

but doing the same with cyberduck gets Connection Failed:  Broken pipe. The connection attempt was rejected. The server may be down, or your network may not be properly configured.

1. Open Cyberduck browser
2. Click on test connection
3. Connection immediately fails and countdown 10 sec to retry
4. See Connection failed dialog box

**Expected behavior**
successful login to remote, and remote browser to to display directory
 
**Screenshots**
configuration
<img width="522" alt="Screenshot 2024-09-22 at 11 55 45 AM" src="https://github.com/user-attachments/assets/ad7cc160-8689-4804-bbd1-e6395377e853">

**Desktop (please complete the following information):**
 - OS: Mac OS Sonoma (M1 processor)
 - Version 14.7

**Log Files**
Please include the [log file]
[cyberduck.log](https://github.com/user-attachments/files/17090305/cyberduck.log)

**Additional context**

**Comment by vincentluciani** [comment]
Hello,
Is it related to the issue I raised (https://github.com/iterate-ch/cyberduck/issues/16366)?
Looking at your screenshot, I see the url (value in blue with hyperlink: sftp://192.168.146.200) does not contain the user name (should be sftp://test@192.168.146.200 ?). That would explain why the connection functioned properly with the command line (sftp test@192.168.146.200) versus the connection with Cyberduck (sftp 192.168.146.200)

**Comment by wnjohnson3** [comment]
Yes, exactly.  I did not notice that.  In past versions the username was automatically added to the url.  Now I can’t figure out how to make that happen.

**Comment by dkocher** [comment]
The log indicates a broken pipe error connecting to `192.168.146.200`.  Please share the verbose output for a successful connection with `ssh -v` from Terminal.app
